packages/react-docsite-components/src/components/Page/Page.types.ts

```diff
@@ -112,6 +112,22 @@ export interface IPageProps<TPlatforms extends string = string> {
 
   /** Currently selected platform. */
   platform?: TPlatforms;
+
+  /**
+   * The available versions whose documentation is presented on the website.
+   */
+  versionOptions?: IContextualMenuItem[];
```

Comment:
Wondering if it would be better to bundle these under a single property. Also if it might be better to just pass the list of versions (or possibly key/text pairs) and let the dropdown rendering code handle converting to menu items, and have the callback just take the chosen version number as a parameter.